vassal-app/src/main/java/VASSAL/tools/imports/adc2/SymbolSet.java

```diff
@@ -586,7 +587,7 @@ protected void readPermutationFile(File f) throws IOException {
     sdx = action.getCaseInsensitiveFile(sdx, f, false, null);
     if (sdx != null) { // must reorder image indeces
 
-      try (Reader fr = new FileReader(sdx);
+      try (Reader fr = new FileReader(sdx, StandardCharsets.UTF_8);
```

Comment:
I should ask Michael if he remembers anything about the charset used in Adc2. It's likely either 7-bit ASCII or Windows-1252.